### PR TITLE
Add Swift templates

### PR DESCRIPTION
**Pull request by kabiroberai**
This pull request adds `application_swift` and `tool_swift` templates based off their Objective-C counterparts. It shouldn't be merged until [libswift](https://github.com/kabiroberai/libswift) is added to a default repo.

### CODE REVIEW

File: ios/application_swift/RootViewController.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+	override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		return objects.count
+	}
+
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		let cell = tableView.dequeueReusableCell(withIdentifier: "Cell", for: indexPath)
+
+		let date = objects[indexPath.row]
+		cell.textLabel!.text = date.description
```
Comment (kabiroberai) [inline ios/application_swift/RootViewController.swift]: Would this be better if it were `cell.textLabel?.text` instead? cc @kirb

Comment (kirb) [reply ios/application_swift/RootViewController.swift]: Personally not a fan of using `?` in situations where it should be an error that the optional is nil. What does Apple’s master-detail app template do?

Comment (kabiroberai) [reply ios/application_swift/RootViewController.swift]: Almost everything is force unwrapped in Apple's template, including `window` in AppDelegate.swift (whereas this template uses optional chaining for `window`)

Comment (kirb) [reply ios/application_swift/RootViewController.swift]: I’d make it all `!` then. Of course keep the optionality of the dictionary arg on application: didFinishLaunchingWithOptions: so it matches the UIApplicationDelegate protocol.

**Comment by kabiroberai** [comment]
@kirb can we merge this now?

**Pull request merged**